project/scripts/bootstrapCmdTests.bat

```diff
@@ -0,0 +1,125 @@
+@echo off
+
+rem ##########################################################################
+rem ## This batch file is based on shell script project/scripts/bootstrapCmdTests
+
+setlocal enabledelayedexpansion
+
+rem only for interactive debugging
+set _DEBUG=0
+
+rem ##########################################################################
+rem ## Environment setup
+
+set _BASENAME=%~n0
+
+set _EXITCODE=0
+
+for %%f in ("%~dp0..\..") do set _ROOT_DIR=%%~sf
+set _SCRIPTS_DIR=%_ROOT_DIR%\project\scripts
+set _BIN_DIR=%_ROOT_DIR%\bin
+
+if not defined __COMMON__ (
+    if %_DEBUG%==1 echo [%_BASENAME%] call %_SCRIPTS_DIR%\common.bat
+    call %_SCRIPTS_DIR%\common.bat
+    if not !_EXITCODE!==0 goto end
+)
+
+rem ##########################################################################
+rem ## Main
+
+rem # check that benchmarks can run
+if %_DEBUG%==1 echo [%_BASENAME%] "%_SBT_CMD%" "dotty-bench/jmh:run 1 1 tests/pos/alias.scala"
+call "%_SBT_CMD%" "dotty-bench/jmh:run 1 1 tests/pos/alias.scala"
+if not %ERRORLEVEL%==0 ( set _EXITCODE=1& goto end )
+
+rem # The above is here as it relies on the bootstrapped library.
+call "%_SBT_CMD%" "dotty-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
+call "%_SBT_CMD%" "dotty-bench-bootstrapped/jmh:run 1 1 -with-compiler compiler/src/dotty/tools/dotc/core/Types.scala"
```

Comment:
One naive question: we don't need to use windows separator for source file, like `tests\pos\alias.scala`?

Reply:
Path `tests/pos/alias.scala` is given **inside** a sbt command wich supports both kinds of file separators.